oat-sa/extension-tao-outcomeui
# Fix/NEC-83/preview math entry results

**Related to task:** 
https://oat-sa.atlassian.net/browse/NEC-83
https://oat-sa.atlassian.net/browse/NEC-82
**Description:**  Fix preview for math items on results page
**How to test:** 
- Create math item with selected "use expression with gaps" option
- Add to test, delivery, pass the test (Answer should be some complex formula)
- On result page click on preview to check the result

**Unit tests cli:** `vendor/bin/phpunit ./taoOutcomeUi/test/integration/helper/ResponseVariableFormatterTest.php `
